lib/Mail/DKIM/ARC/Verifier.pm

```diff
@@ -193,6 +193,11 @@ sub add_signature {
 
     return if $self->{result};    # already failed
 
+    # Set verification time if we have one
+    if ($self->{verify_time}) {
+        $signature->{_verify_time} = $self->{verify_time};
```

Comment:
I know it's only convention, but I strongly dislike assignment on a `_private` key, and from a quick look there's nowhere else in the codebase where assignment to a private key is done on anything but `$self`

Reply:
Have created a setter and used that instead

Reply:
Cheers.  I know it's a small thing, but I really do think making it clear what data is encapsulated and isn't is a distinction worth upholding